examples/sticky.ads.amp.html

```diff
@@ -160,7 +159,7 @@
     </div>
   </header>
   <main role="main">
-  <amp-sticky-ad layout="nodisplay">
+  <amp-sticky-ad layout="container">
```

Comment:
I add `toggle(this.element, true)` to `buildCallback()`, unfortunately I still see blink